tests/MyGame/MonsterExtra.py

```diff
@@ -217,19 +217,6 @@ def InitFromObj(cls, monsterExtra):
         x._UnPack(monsterExtra)
         return x
 
-    def __eq__(self, other):
```

Comment:
Looks like this was deleted on accident? Did you rebase recently?

Reply:
I did a rebase 2 commits before the last one. Should I rebase again?

Reply:
it seems that these are deleted by the generate_code.py, I tried running this scripts on master, and it does emit these changes.

Reply:
So I manually revert these changes, but for some reason, every time I ran scripts/generate_code.py, these go, dart, swift, python files got changed. And this __eq__ function is also deleted.

Reply:
I figured out what I did wrong, after my rebasing, I just ran generate_code.py without rerunning `make flatc` first, so it was generating code with the old binary before my rebasing. Sorry for the back and forth commits, I think this time should be all good.